scripts/zones/Arrapago_Reef/MobIDs.lua

```diff
@@ -0,0 +1,6 @@
+-- Medusa
+Medusa    = 16998862;
+ExonOne   = 16998863;
```

Comment:
I know I can do that. I wanted to do it this way to make it easier for anyone else coding it that may not understand that.